src/content-helper/editor-sidebar/title-suggestions/store.ts

```diff
@@ -290,15 +290,15 @@ export const TitleStore = createReduxStore( 'wp-parsely/write-titles', {
 			}
 			return undefined;
 		},
-		getTone( state: TitlesState ): ToneProp {
+		getTone( state: TitlesState ): ToneProp|null {
```

Comment:
This was the easiest way to support settings without changing a lot of things, but please let me know if you think there's a better alternative.

Reply:
In fact, I think that now we don't even need the store anymore. Thoughts?

Reply:
OK, I'll try remove the store as part of another PR to see what happens 🙂

Reply:
Done in #2122.